setup.py

```diff
@@ -45,5 +51,13 @@
         "defconQt.representationFactories",
         "defconQt.util",
     ],
-    package_dir={"": "Lib"}
+    scripts=["trufont"],
```

Comment:
I can't seem to find the right module attribute to import: 

entry_points={
        'console_scripts': [
            "trufont = defconQt:main"
        ]

Any ideas?